src/main/java/org/sonarqube/A.java

```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
   public String foo() {
+    message = null;
+    message.length();
+
     return message;
```

Comment:
###AWASt6RZg9QtoM543Hc_###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_19@9
A "NullPointerException" could be thrown; "message" is nullable here.

Reply:
===RESOLVED===
This issue has been resolved by a user in SonarQube

Reply:
simulating user replying to bot's comments

Reply:
simulating a non standard comment posted by bot

Reply:
and a 2nd one, just because